Neos.Neos/Classes/Http/ContentDimensionDetection/BackendUriDimensionPresetDetector.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Neos\Neos\Http\ContentDimensionDetection;
```

Comment:
```suggestion
declare(strict_types=1);

namespace Neos\Neos\Http\ContentDimensionDetection;
```

All newly added files must contain the `strict_types=1` declaration.